ArduPlane/quadplane.cpp

```diff
@@ -1092,8 +1074,11 @@ bool QuadPlane::is_flying_vtol(void) const
         // assume that with no motor outputs we're not flying in VTOL mode
         return false;
     }
-    if (motors->get_throttle() > 0.01f) {
-        // if we are demanding more than 1% throttle then don't consider aircraft landed
+    if (motors->get_throttle() > 0.01f && !motors->limit.throttle_lower) {
```

Comment:
This line is a change in behavior, we no longer return if the throttle is above 1% but it is on the lower limit. This heuristic comes from the old `is_flying`